defaults/php_config_ref.yml

```diff
@@ -0,0 +1,48 @@
+---
+# define parameters depending for the php version installed
+# for any new version, only specify keys with values different from defaults
+# defaults should work for php > 7.3
+defaults:
+  php_dir: "/etc/php/{{ php_ver }}"
+  php_bin: "php-fpm{{ php_ver }}"
+  php_pkg_apcu: "php{{ php_ver }}-apcu"
```

Comment:
You can replace php_pkg_apcu by:

if version < 7.3 it is php-apcu otherwise it is php{{ php_ver }}-apcu.

Reply:
Actually it  also works for php7.4